makefile

```diff
@@ -66,13 +62,13 @@ generate-proto:
 	@mkdir -p ./interfaces/
 	@protoc --go_out=./interfaces/ --go-grpc_out=./interfaces/ -I ./contracts/proto ./contracts/proto/*/**/*.proto
 
-# Build all service factory plugins
-plugins: aws-plugin gcp-plugin dev-plugin
-	@echo Done.
+# # Build all service factory plugins
+# plugins: aws-plugin gcp-plugin dev-plugin
+# 	@echo Done.
 
-test-plugins: install-tools
```

Comment:
Need to restore this, these are the tests for the plugins (both static and pluggable membrane)

Reply:
Done